Bottom: 6c62c310cec99f8c3d4d8f0b1d129fd8772720a5
Top:    bd2a39253a0b01f7f213d9bb2e5a4c9dea557e70
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 16:13:27 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/daemon/direct.c b/daemon/direct.c
index e004e50..9d8bcc3 100644
--- a/daemon/direct.c
+++ b/daemon/direct.c
@@ -26,12 +26,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <sys/ioctl.h>
-#include <sys/mount.h>
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <sys/time.h>
 #include <sys/poll.h>
+#include <sys/mount.h>
 #include <sys/vfs.h>
 #include <sched.h>
